### PR TITLE
[feature](mtmv)add Job and task tvf

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

add:
select * from jobs("type"="mv");
select * from tasks("type"="mv");
select * from jobs("type"="insert");
select * from tasks("type"="insert");

add check priv for mv_infos("database"="xxx");

change JobType MTMV==>MV

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/tablefunction/JobsTableValuedFunction.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        }
+        TJobsMetadataParams jobMetadataParams = params.getJobsMetadataParams();
+        String type = jobMetadataParams.getType();
+        JobType jobType = JobType.valueOf(type.toUpperCase());
+        if (jobType == null) {
+            throw new AnalysisException("Invalid job metadata query");
+        }
+        if (JobType.MTMV == jobType) {
+            return MTMVJob.COLUMN_TO_INDEX.get(columnName.toLowerCase());
+        } else {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/tablefunction/JobsTableValuedFunction.java:86]: else if.
and the last else, throw exception

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run feut

**Review by morningman — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Comment by morningman** [comment]
@CalvinKirs please help to add doc and test case in next PR

**Comment by CalvinKirs** [comment]
> @CalvinKirs please help to add doc and test case in next PR↳

will do.

**Pull request merged**